backend/authors/views.py

```diff
@@ -0,0 +1,70 @@
+from django.shortcuts import render
+from django.http import HttpResponse, JsonResponse
+from rest_framework.pagination import PageNumberPagination
+from rest_framework.parsers import JSONParser
+from rest_framework.generics import GenericAPIView
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from authors.models import Author 
+from authors.serializers import AuthorSerializer
+from authors.pagination import PaginationHandlerMixin
+
+class AuthorView(GenericAPIView):
+    pagination_class = PageNumberPagination
+
```

Comment:
I'm surprised if this works on your end, I can't actually get the following piece of code to run on my machine without setting a default serializer and a default queryset. 

So I would need to add this right under line 13
```
serializer_class = AuthorSerializer
queryset = Author.objects.all()
```